policies/access.md

```diff
@@ -0,0 +1,41 @@
+# Github Access Management Policy
+
+## Teams, not Individuals
+
+Access to repositories will be granted only to Github teams, not to individuals. A repo may have any number of teams added to it, with the appropriate access levels.
+
+## Team Structure
+
+Teams will be nested, using parent/child relationships, as needed. A child team will implicitly have any privileges its parent team does, so, the most deeply nested team should be the one with the most privileges.
+
+### Top-level teams
+
+Note: this list is intentionally not exhaustive.
+
+- [TAC](https://github.com/orgs/ossf/teams/tac)
+    This team is for current TAC members. Individuals should be added or removed from this team to reflect current membership.
+- [Staff](https://github.com/orgs/ossf/teams/staff)
+    This team is for OpenSSF staff members. Individuals on the top-level team are there as a catch-all, but may be moved to subteams as the need arises.
+  - [PMs](https://github.com/orgs/ossf/teams/pms)
+      This teams is for PMs, who often need Maintain or Admin access on repos.
+  - [Marketing](https://github.com/orgs/ossf/teams/marketing)
+- [Working Groups](https://github.com/orgs/ossf/teams/working-groups)
+    This is the parent team for Working Groups. Every WG should have a subteam contained within this one. All WG subteams must start with `wg-` for consistency.
+- [SIGs](https://github.com/orgs/ossf/teams/sigs)
+    This is the parent team for SIGs. Every SIG should have a subteam contained within this one. All SIG subteams must start with `sig-` for consistency.
+- [Projects](https://github.com/orgs/ossf/teams/projects)
+    This is the parent team for projects. Every project (eg scorecard, AO) should have a subteam contained within this one.
+    Teams for individual repositories go under here, which start with `repo-`, but team names may otherwise be unconstrained.
+
+## Github Org Membership
+
+ Membership in the Github org should be freely given - it inherently confers no permissions or privileges, only a badge on the user's profile if they enable it - and it _does_ allow for easier team management. Someone should only be removed from the org in extreme circumstances where their association with OpenSSF would be problematic, and people should be encouraged to remain in the org in perpetuity.
```

Comment:
This feels different than many open source projects. Especially for a relatively new and also broad umbrella foundation, I worry that this freely given perpetual membership even if only a weak badging, might encourage confusion on what is or is not officially part, sponsored by, under consideration within the auspices of the foundation. That type of confusion and contention has already been a bit of a problem for the OpenSSF org. 

I personally would prefer a simple and collaborative open membership. But I also want clarity of mission and propose and that to be easily discovered and difficult to confuse. 

Perhaps this is less of an issue if leadership (ie: certain team memberships in the org?) roles are clearly marked. But GitHub team memberships tend to be opaque. I’m not sure if there is maybe an org setting which would allow team members be viewed by non org members.  Or if not…alternatively if anybody can get org membership, I guess anybody could ask for org membership to be able to see team membership?  Or GH team members defined by in repo yaml configs?  Still not great UX though.

Reply:
@tpepper OpenSSF is not itself an open source project, and this is how TC39 operates, which I think is more analogous.

I agree that it's not great UX that the only way to be on a team is to be in the org, but it seems a worthy tradeoff for me.